include/aspect/mesh_refinement/vof_interface.h

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
The two separate implementations were an artifact of retaining a working earlier mesh refinement method while implementing a better approach. Since I have implemented a correct plugin based on flagging cells, I have removed the older method.